### PR TITLE
Task01 was completed by Muhammadsidiqjon

### PR DESCRIPTION
**Pull request by Nasa-133**
This is task about carshop

### CODE REVIEW

File: Muhammadsiddiq/oop_task1_try2.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+        if tolov =="exit":
+            print("sizni ko'rganimdan xursandman ")
+            break
+        else:
+            print("Noto'g'ri kiritingiz, qayta urinib ko'ring.")
+
+else:
+    print("Tashrifingiz uchun rahmat! Sizni xarid qilishingiz uchun yana kutib qolamiz")
+
+
```
Comment (Shaxzodbek16) [inline Muhammadsiddiq/oop_task1_try2.py:182]: add main function to you script.

File: Muhammadsiddiq/oop_task1_try2.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+
+
+
```
Comment (Shaxzodbek16) [inline Muhammadsiddiq/oop_task1_try2.py:186]: add `if __name__ == '__main__':`

File: Muhammadsiddiq/oop_task1_try2.py [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            print(f"Bizda mavjud bo'lgan mashina turlari \n1.{car_max.name}={car_max.price}  \n2.{sienna_hybrid.name}={sienna_hybrid.price} \n3.{odyssey.name}={odyssey.price}\n")
+            break
+        elif turi == 'Suv' or turi == '4':
+            my_car["Type"] = "Suv"
+            print(f"Bizda mavjud bo'lgan mashina turlari \n1.{hyundai_tucson.name}={hyundai_tucson.price}  \n2.{mazda_cx.name}={mazda_cx.price} \n3.{kia_sportage.name}={kia_sportage.price}\n")
+            break
+        else:
+            print("Noto'g'ri tanlov. Qayta urinib ko'ring.")
+
+def select_model(my_car):
```
Comment (IslomboevBotir) [inline Muhammadsiddiq/oop_task1_try2.py:44]: This func too should be in class

File: Muhammadsiddiq/oop_task1_try2.py [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+                break
+            elif model == "3":
+                my_car["Model"] = kia_sportage.name
+                my_car["Price"] = kia_sportage.price
+                print(f"Sizning mashinangiz {kia_sportage.name}")
+                break
+        else:
+            print("Noto'g'ri tanlov. Qayta urinib ko'ring.")
+
+def select_color(my_car):
```
Comment (IslomboevBotir) [inline Muhammadsiddiq/oop_task1_try2.py:114]: That func too should be in class

**Pull request closed without merging**